Tests/BridgeComponentTests.swift

```diff
@@ -223,6 +224,6 @@ class BridgeComponentTest: XCTestCase {
     }
 }
 
-private extension TimeInterval {
+extension TimeInterval {
     static let expectationTimeout: TimeInterval = 5
 }
```

Comment:
Let's move this to a new file if it is shared outside of this one.

Reply:
Done.